solutions/landing-zone/README.md

```diff
@@ -188,9 +188,16 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
     
     Before deploying with `kpt` you will need to add `constraints.yaml` to the `.krmignore` file. This is due to needing to have the `constraintstemplate` resources deployed into the instance before the policy `constraint` can be deployed. Once the `constrainttemplates` have been deployed you can remove `constraints.yaml` from the `.krmignore` file and redeploy. This is not needed with either gitops deployment options.
 
+    Before running the `kpt` commands for the first time you will need to initialize the `kpt` package locally. You will only need to do this once after you initially get the `kpt` package
```

Comment:
LGTM
I'll need to test this locally

Reply:
Will hold off on merging until you give the all clear.

Reply:
I might not get to this testing today, reworking a couple of my kcc lz orgs.  The change is good and critical - I can side test after it is merged.

Good to merge